.github/workflows/appimage-workflow.yml

```
@@ -0,0 +1,139 @@
+name: AppImage Build
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - .gitignore
+      - "**/*.md"
+      - .clang-format
+      - debian-packager/
+      - bin/PCSX2_keys.ini.default
+      - "pcsx2/PAD/Windows/**"
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - .gitignore
+      - "**/*.md"
+      - .clang-format
+      - debian-packager/
+      - bin/PCSX2_keys.ini.default
+      - "pcsx2/PAD/Windows/**"
+
+jobs:
+  build:
+    strategy:
+      # Prevent one build from failing everything (although maybe those should be included as experimental builds instead)
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            platform: x86
+            compiler: gcc
+            detail: -appimage
+            experimental: true
+          - os: ubuntu-18.04
```

Might want to use the latest LTS for consistency.
```suggestion
          - os: ubuntu-20.04
            platform: x86
            compiler: gcc
            detail: -appimage
            experimental: true
          - os: ubuntu-20.04
```